docs/reference/ilm/ilm-with-existing-indices.asciidoc

```diff
@@ -0,0 +1,409 @@
+older periodic indices can be deleted.
+
+[[ilm-reindexing-into-rollover]]
+=== Reindexing via {ilm-init}
+
+NOTE: The examples in this section assume daily indices as set up in
+<<ilm-with-existing-indices,the previous section>>.
+
+In some cases, it may be useful to reindex data into {ilm-init}-managed indices.
+This is more complex than simply attaching policies to existing indices as
+described in <<ilm-with-existing-periodic-indices,the previous section>>, and
+requires pausing indexing during the reindexing process. However, this technique
+may be useful in cases where periodic indices were created with very small
+amounts of data leading to excessive shard counts, or for indices which grow
+steadily over time, but have not been broken up into time-series indices leading
+to shards which are much too large, situations that cause significant
+performance problems.
+
+Before getting started with reindexing data, the new index structure should be
+set up. For this section, we'll be using the same setup described in
+<<ilm-with-existing-indices,{ilm-imit} with existing indices>>.
+
+First, we'll set up a template with rollover, and can include any additional
+phases required. For simplicity, we'll just use rollover:
+
+[source,js]
+-----------------------
+PUT _ilm/policy/sample_policy
+{
+  "policy": {
+    "phases": {
+      "hot": {
+        "actions": {
+          "rollover": {
+            "max_age": "7d",
+            "max_size": "50G"
+          }
+        }
+      }
+    }
+  }
+}
+-----------------------
+// CONSOLE
+// TEST[continued]
+
+And now we'll update the index template for our indices to include the relevant
+{ilm-init} settings:
+
+[source,js]
+-----------------------
+PUT _template/mylogs_template
+{
+  "index_patterns": [
+    "ilm-mylogs-*" <1>
+  ],
+  "settings": {
+    "number_of_shards": 1,
+    "number_of_replicas": 1,
+    "index": {
+      "lifecycle": {
+        "name": "mylogs_condensed_policy", <2>
+        "rollover_alias": "mylogs" <3>
+      }
+    }
+  },
+  "mappings": {
+    "properties": {
+      "message": {
+        "type": "text"
+      },
+      "@timestamp": {
+        "type": "date"
+      }
+    }
+  }
+}
+-----------------------
+// CONSOLE
+// TEST[continued]
+<1> The new index pattern has a prefix compared to the old one, this will
+    make it easier to reindex later
+<2> The name of the policy we defined above
+<3> The name of the alias we'll use to write to and query
+
+And create the first index with the alias specified in the `rollover_alias`
+setting in the index template:
+
+[source,js]
+-----------------------
+PUT ilm-mylogs-000001
+{
+  "aliases": {
+    "mylogs": {
+      "is_write_index": true
+    }
+  }
+}
+-----------------------
+// CONSOLE
+// TEST[continued]
+
+At this point, new indexing should be paused to ensure that all existing data is
+reindexed properly. Adding new data to existing indices during this process can
```

Comment:
"existing" is not the right word here since technically now `ilm-mylogs-000001` is an "existing" index and it's totally okay to index into it during this process